### PR TITLE
Add gradations for autoscaling

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Adds the steps for each disk size with autoscaling.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes https://github.com/timescale/docs/issues/630

### CODE REVIEW

File: cloud/scaling-a-service.md [hunk 1]
```diff
@@ -103,6 +103,25 @@ longer, and in some cases this could create a delay longer than six hours.
...
+*   400&nbsp;GB
+*   500&nbsp;GB
+*   600&nbsp;GB
+*   700&nbsp;GB
+*   800&nbsp;GB
+*   900&nbsp;GB
+*   1&nbsp;TB
+*   1.5&nbsp;TB
+*   2&nbsp;TB
+*   10&nbsp;TB
```
Comment (lukas-bernert) [inline cloud/scaling-a-service.md:123]:
```suggestion
*   10&nbsp;GB
*   25&nbsp;GB
*   50&nbsp;GB
*   75&nbsp;GB
*   100&nbsp;GB
*   125&nbsp;GB
*   150&nbsp;GB
*   175&nbsp;GB
*   200&nbsp;GB
*   225&nbsp;GB
*   250&nbsp;GB
*   275&nbsp;GB
*   300&nbsp;GB
*   325&nbsp;GB
*   350&nbsp;GB
*   375&nbsp;GB
*   400&nbsp;GB
*   425&nbsp;GB
*   450&nbsp;GB
*   475&nbsp;GB
*   500&nbsp;GB
*   600&nbsp;GB
*   700&nbsp;GB
*   800&nbsp;GB
*   900&nbsp;GB
*   1&nbsp;TB
*   1.5&nbsp;TB
*   2&nbsp;TB
*   2.5&nbsp;TB
*   3&nbsp;TB
*   4&nbsp;TB
*   5&nbsp;TB
*   6&nbsp;TB
*   7&nbsp;TB
*   8&nbsp;TB
*   9&nbsp;TB
*   10&nbsp;TB
*   12&nbsp;TB
*   14&nbsp;TB
*   16&nbsp;TB
```

**Comment by Loquacity** [comment]
Thanks @lukas-bernert ! The list has gotten long, so I've moved it to after the procedure to help the flow.

**Review by lukas-bernert — APPROVED** [review]

**Review by emizero — APPROVED** [review]
Adding it to the end definitely helps the flow.  
As a future consideration would be listing them in 4-5 columns of equal length.  Don't know if that is possible.

**Comment by Loquacity** [comment]
> Adding it to the end definitely helps the flow. As a future consideration would be listing them in 4-5 columns of equal length. Don't know if that is possible.

That's a @jacobprall question 😉 

Having said that, I think I prefer it in a single list, only because our centre reading pane can be quite narrow, and sideways scrolling sucks.

**Pull request merged**